metadata-ingestion/tests/integration/cassandra/setup/cassandra.yaml

```diff
@@ -0,0 +1,1827 @@
+# ---------------------------------------------------------------------------------------- #
+# NOTE:
+# This yaml has been copied from the official Cassandra Docker image:
+# (https://github.com/datastax/docker-images/blob/master/config-templates/DSE/6.0.0/cassandra.yaml)
+# 
+# Key interests are in setting the AllowAllNetworkAuthorizer,
+# and in enabling materialized view feature.
+# ---------------------------------------------------------------------------------------- #
+
```

Comment:
If possible, this would be great  -> https://github.com/datahub-project/datahub/pull/9680/files#r1460632601

Reply:
I have removed the commented part